android/src/main/java/com/swmansion/reanimated/nodes/AlwaysNode.java

```diff
@@ -1,12 +1,13 @@
 package com.swmansion.reanimated.nodes;
 
 import com.facebook.react.bridge.ReadableMap;
+import com.swmansion.reanimated.MapUtils;
 import com.swmansion.reanimated.NodesManager;
 
 public class AlwaysNode extends Node implements FinalNode {
   public AlwaysNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
     super(nodeID, config, nodesManager);
-    mNodeToBeEvaluated = config.getInt("what");
+    mNodeToBeEvaluated = MapUtils.getInt(config, "what", "Argument passed to always node might be of wrong type. NodeID: " + nodeID );
```

Comment:
Still, `"what"` means nothing for a developer. It's the value which has to be set and the name `what` does not imply it. 


Also, `might be of wrong type` is not true. It actually *is* of a wrong type or is missing. Be confident in this error message.

Reply:
"what" is a key we are retrieving here. I do not specify anything about the argument passed right here as there is only one. In cases where there are two or more, I explicitly state it is first, second, etc. based on how we pass it in JavaScript

I agree the message is not clear enough.

Reply:
what?